.deepsource.toml

```diff
@@ -0,0 +1,15 @@
+version = 1
+
+test_patterns = ["testing/**/test_*.py"]
+
+exclude_patterns = [
+    "changelog/**",
+    "doc/en/**"
+]
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
```

Comment:
we're not using this vendor so I don't think we should add this file

from my previous experience with deepsource it is more of an annoyance than a benefit -- in particular there is no way to run it locally